src/components/dashboard/immunizationStatus/immunizationStatusCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Flex, Icon, Stack, Text } from '@chakra-ui/react';
+import React, { FC } from 'react';
+import { ImmunizationStatus } from '../../../core/models/immunizationStatus';
+
+interface ImmunizationStatusCardProps {
+  status: ImmunizationStatus;
+}
+
+export const ImmunizationStatusCard: FC<ImmunizationStatusCardProps> = ({
+  status,
+}) => {
+  let currentStatus: string = 'default';
```

Comment:
This is unused atm but will be needed in the future, so I would say it's fine. But regarding future usage, it should probably not be a string but of type ImmunizationStatus?